src/app/features/register/register.component.html

```diff
@@ -1,59 +1,102 @@
-<main>
-  <fieldset>
-    <app-input-field
-      [labelFor]="'username'"
-      [labelClassName]="'label-username'"
-      [labelText]="'Username'"
-      [inputType]="'text'"
-      [inputId]="'input-username'"
-      [inputName]="'username'"
-      [inputPlaceholder]="'e.g. tango123'"
-      [inputClassName]="'input-username'"
-      (inputChange)="onChange($event)"
-    >
-    </app-input-field>
+<fieldset>
+  <form [formGroup]="register" (ngSubmit)="onSubmit()" id="registerForm">
+    <div class="label-input">
+      <div>
+        <label id="label-email" for="input-email"> Email: </label>
+        <span role="alert" class="error">
```

Comment:
`role="alert"` will cause the Voice Over to announce whatever text it is inside the tag.